atlasdb-commons/src/main/java/com/palantir/common/time/NanoTime.java

```diff
@@ -14,26 +14,32 @@
  * limitations under the License.
  */
 
-package com.palantir.leader.lease;
+package com.palantir.common.time;
 
 import java.time.Duration;
 import java.util.Objects;
 import java.util.concurrent.locks.LockSupport;
 
-import com.google.common.annotations.VisibleForTesting;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
 
 public final class NanoTime implements Comparable<NanoTime> {
```

Comment:
out of curiosity, how come this class isn't immutables? but all the other classes are?

Reply:
yeahhhh, this should use immutables... sorry, my bad

Reply:
(no need to do this in this PR; can be a little change on its own)